Makefile.am

```diff
@@ -35,7 +35,8 @@ SUBDIRS = \
 	src/tools/perf \
 	src/tools/profile \
 	test/apps \
-	test/examples
+	test/examples \
+	man
```

Comment:
Open question - should be here or go under ```doc/man```

Reply:
If we went the doxygen route then it could end up in doc/man.  Otherwise worried about it getting trampled on.